### ISSUE TITLE
ProbeError: unrecognized file format

### ISSUE THREAD

**Issue by yolpsoftware** [nodeca/probe-image-size#61]
Same error as in #57, but different image:

Stacktrace:

```
   at ParserStream.parserEnd (my-repo\node_modules\probe-image-size\stream.js:39:28)
    at Object.onceWrapper (events.js:421:28)
    at ParserStream.emit (events.js:315:20)
    at endReadableNT (_stream_readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```
Code used:

```
import probeImageSize from 'probe-image-size';

const result = await probeImageSize("https://upload.wikimedia.org/wikipedia/commons/8/82/Scotland_in_the_UK_and_Europe.svg");
```

Environment: node 14.15.1 on Windows 10
probe-image-size: 7.1.1

Other images that have the same problem:

```
https://upload.wikimedia.org/wikipedia/commons/3/3a/Locator_map_of_Azores_in_EU.svg
https://upload.wikimedia.org/wikipedia/commons/3/3e/EU-Portugal_with_Madeira_circled.svg
https://upload.wikimedia.org/wikipedia/commons/5/55/Gibraltar_location_in_Europe.svg
```

**Comment by puzrin** [comment]
Thanks for report.

Reason is different. Sources use namespaced format: `<svg:svg ...>` (i never seen it before)

**Comment by puzrin** [comment]
Released 7.2.0

**Comment by yolpsoftware** [comment]
Thanks, works perfectly.
